.pre-commit-config.yaml

```diff
@@ -47,7 +47,7 @@ repos:
     hooks:
       - id: pixi-install
         name: pixi-install
-        entry: pixi install
+        entry: pixi install --manifest-path /pyproject.toml
```

Comment:
Ah, just saw the CI logs -- I think this one will be an issue b/c of the need for pre-commit hooks to work both within a devcontainer and through a local pixi installation. What are your thoughts on the best way to support both? Move back to $CWD/pyproject.toml?

Reply:
That would not work. If I just change this back then pre-commit will fail locally and I cannot commit. Let ma take a closer look into thsi.

Reply:
@ericmjl I messed up several things in my local permissions, I am gonna close this PR. reclone and then reopen it, sorry for the inconvenience,